### ISSUE TITLE
PPYOLOE+x蒸馏l在自己数据集上效果很差的原因

### ISSUE THREAD

**Issue by Hanjiangli** [PaddlePaddle/PaddleDetection#7954]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

![eb55886d2b4ed535257d85e8553c928](https://user-images.githubusercontent.com/107346316/225315682-3abb98ff-2fde-4653-9d89-1d54d4d0abbb.png)
我已经在自己数据集（大概4000多张图片，尺寸都是320*240）单独训练了ppyoloe+的x模型（跑了80e），然后蒸馏的时候加载这个作为teacher模型的预训练模型，上图是x蒸馏l的训练，效果很差（如果单独用l模型训练，map在10e的时候能达到0.85+），请问是我某些参数没设置好吗？运行命令如下图，bs是8，以上训练都是在aistudio平台的V100单卡环境下进行。
![fb82dc6cf43c5801fdffe2b9a0eb592](https://user-images.githubusercontent.com/107346316/225317534-214e3f0f-0ef7-4a66-8ecc-2dc63ae8a236.png)

**Comment by fangfangssj** [comment]
我猜你的数据集是完全模型组线上赛的数据集，我尝试使用l模型蒸馏m模型的过程中也遇到这样的状况😂

**Comment by nemonameless** [comment]
logits_loss 这么大不正常，建议发下第0epoch开头 loss log

**Comment by Hanjiangli** [comment]
![image](https://user-images.githubusercontent.com/107346316/225498977-6f9336b7-4c33-4c89-9422-09b122483e63.png)
![image](https://user-images.githubusercontent.com/107346316/225499394-deed2928-7569-4025-8fdc-7fcd45084cf4.png)

**Comment by Nirvana93** [comment]
我也是遇到了一样的问题，不仅仅在x蒸馏l上，在其他尺寸的模型蒸馏方案上有
